spec/features/bars/user_updates_bar_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature "user can update a bar's information", %{
+  As an authenticated user
+  I want to update an bar's information
+  So that I can correct errors or provide new information
+} do
+
+  #  Acceptance Criteria
+  # [x] User can navigate to a page and
+  #     change bar’s details except name and address
+  # [x] User must be logged in to change bar details
+  # [x] User views form prepopulated with bar details
+  # [X] User is redirected to bar detail page and can see their changes
+
+  scenario 'user can navigate to a bar\'s edit page' do
+    bar = FactoryGirl.create(:bar)
+    user = FactoryGirl.create(:user)
```

Comment:
We decided not to give a bar an  "owner" and to allow the details to be edited at will by any user (admin or no).